1558671257326	Sync.LogManager	DEBUG	Flushing file log
1558671257360	Sync.LogManager	DEBUG	Log cleanup threshold time: 1557807257360
1558671257426	Sync.LogManager	DEBUG	Done deleting files.
1558671258261	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1558671259478	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1558671267007	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1558671572987	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1558671572990	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1558671572995	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1558671573986	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1558671575246	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1558671575246	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1558671575255	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Linux x86_64) FxSync/1.69.0.20190521202118.desktop
1558671575255	Sync.Service	INFO	Starting sync at 2019-05-23 22:19:35 in browser session Igju9bUWkDxV
1558671575256	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1558671575261	Sync.Status	INFO	Resetting Status.
1558671575261	Sync.Status	DEBUG	Status.service: error.sync.failed => success.status_ok
1558671575282	Sync.Resource	WARN	GET request to https://sync-548-us-west-2.sync.services.mozilla.com/1.5/84144729/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1558671575282	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1558671575282	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1558671575283	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1558671575284	Sync.SyncScheduler	DEBUG	Sync error count has exceeded 3; enforcing backoff.
1558671575284	Sync.SyncScheduler	DEBUG	Starting client-initiated backoff. Next sync in 3945204 ms.
1558671575284	Sync.SyncScheduler	DEBUG	Next sync in 3945204 ms. (why=client-backoff-schedule)
1558671575289	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
